core/src/test/java/org/apache/iceberg/TestPartitionSpecInfo.java

```diff
@@ -65,11 +65,22 @@ public void cleanupTables() {
     TestTables.clearTables();
   }
 
+  @Test
+  public void testSpecIsUnpartitionedForVoidTranforms() {
```

Comment:
I think this test is good. Do we also need a test for a V1 table whose partitioning was removed? I know that was the main reason this has come up in the past. I only wonder because this test explicitly tests the condition created by that situation so we are probably covered.

Reply:
Yup I can added a test for that as well. Can you point me to the location of that test?

Reply:
@RussellSpitzer Can you give me an example on how to remove partitions from a V1 table? Would love to also add the test.

Reply:
For future reference

```
So in the future you can invoke the V1 Table drop partition columns via either the Iceberg or Spark3 API. In both cases V1 tables is the default (at least at the moment)
// Spark 3 Example
https://github.com/apache/iceberg/blob/master/spark3-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestAlterTablePartitionFields.java#L195-L213
// V1 Spec Example
https://github.com/apache/iceberg/blob/master/core/src/test/java/org/apache/iceberg/TestPartitioning.java#L63
I think it's probably fine to have the test you have so far unless you want to add one to TestPartitioning

TestAlterTablePartitionFields.java
<https://github.com/apache/iceberg|apache/iceberg>apache/iceberg | Added by GitHub

TestPartitioning.java
<https://github.com/apache/iceberg|apache/iceberg>apache/iceberg | Added by GitHub
```

Reply:
Test is here 
https://github.com/apache/iceberg/pull/3059/files#diff-9e7d052ccbda7950f95d5f245052eb249f41b565923959c10e138942632a4292R81-R85